### PR TITLE
make search movies functionality in functions and adapters.

### PR DESCRIPTION
**Pull request by lukisko**
made search for movies, moved logic of functions to adapters and made use of Newtonsoft for json serialisation.

### CODE REVIEW

File: Core/Models/MovieList.cs
Comment (KlausBenislavskis) [inline Core/Models/MovieList.cs:1]: I wouldn't use JsonProperty in the core model, I would create in Adapters a movie model and then map it to core

File: Core/Interfaces/ISearchMovies.cs
Comment (KlausBenislavskis) [inline Core/Interfaces/ISearchMovies.cs:1]: Pascal case for c# methods

**Review by KlausBenislavskis — COMMENTED** [review]
Some comments, nice work

File: Core/Models/MovieList.cs
Comment (lukisko) [reply Core/Models/MovieList.cs:1]: Oh a good catch, forgot to remove json properties after using Newtonsoft.

**Comment by lukisko** [comment]
updated the pull request. I hope I did not forget some additional Pascal cases for methods :smile:

**Pull request merged**